.github/workflows/ghcjs.yml

```
@@ -101,3 +94,14 @@ jobs:
           target-folder: playground
           clean: false
           single-commit: true
+
+      - name: 🔨 Save flake from garbage collection
+        run: nix run .#save-flake
+
+      - name: 🔨 Push flake outputs to cachix
+        if: vars.CACHIX_AUTH_TOKEN != ''
```

Should this be `secrets`?

```suggestion
        if: secrets.CACHIX_AUTH_TOKEN != ''
```